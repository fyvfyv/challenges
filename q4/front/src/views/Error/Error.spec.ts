import { mount } from '@vue/test-utils'
import Component from './index'

describe('Error', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Component, {
      stubs: ['router-link']
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders correctly', () => {
    const wrapper = mount(Component, {
      stubs: ['router-link']
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
