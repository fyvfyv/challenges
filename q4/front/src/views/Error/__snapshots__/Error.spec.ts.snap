// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders correctly 1`] = `
<article
  class="content"
>
  <h1>
    Not found 🙁
  </h1>
   
  <router-link-stub
    to="[object Object]"
  >
    Go to the list
  </router-link-stub>
</article>
`;
